### DISCUSSION TITLE
Disable the Fire summoned/spread by Lightning -- Adjust Lightning Bolt damage

### DISCUSSION THREAD

**Discussion by Ski-z** [ACGaming/UniversalTweaks#194] [Tweak Suggestions]
Vanilla Lightning Bolts also create fire near their strike location. I'd love two tweaks which:

1. Toggle disable lightning causes fires
2. Set Lightning Damage
3. 
Thanks ACG!

**Comment by ACGaming** [comment]
Implemented in 10fa5e8ca5b613204d97b5d2437f78b2707cbd69.
